packages/contracts/scripts/deploy/deploy.ts

```diff
@@ -1,37 +1,29 @@
-import fs from "fs";
-import path from "path";
-import { DEPLOY_FUNCS } from "./manager";
-import { loadEnv } from "../shared";
+import { Deploy_Manager } from "./manager";
 import CommandLineArgs from "command-line-args";
+import { optionDefinitions } from "./definition";
 
 const main = async () => {
-    const cmdArgs = process.argv.slice(2);
-    const name = cmdArgs[0];
-    if (!name) {
-        throw new Error("You MUST put the script name in command arguments at least");
-    }
-
-    const envPath = path.join(__dirname, "../../.env");
-    if (!fs.existsSync(envPath)) {
-        throw new Error("Env file not found");
-    }
-    const env = loadEnv(envPath);
+  const cmdArgs = process.argv.slice(2);
+  const name = cmdArgs[0];
+  if (!name) {
+    throw new Error("You MUST put the script name in command arguments at least");
+  }
 
-  if (!DEPLOY_FUNCS[name]) {
+  if (!Deploy_Manager[name]) {
```

Comment:
Why pascal case here?

Reply:
> Why pascal case here?

The `Deploy_Manager` works as a mapping that returns a function, 
I don't see a strict naming convention followed here.

Could be in camelCase as well.
@pavlovcik